src/main/java/com/google/devtools/build/lib/analysis/config/ConfigurationResolver.java

```diff
@@ -118,7 +118,7 @@
   public static OrderedSetMultimap<DependencyKind, Dependency> resolveConfigurations(
       SkyFunction.Environment env,
       TargetAndConfiguration ctgValue,
-      OrderedSetMultimap<DependencyKind, Dependency> originalDeps,
+      OrderedSetMultimap<DependencyKind, DependencyKey> originalDeps,
```

Comment:
NIt: maybe rename this variable now too?

Reply:
Done. Also in sortResolvedDeps.